tests/test-conf.yaml

```diff
@@ -1,17 +1,20 @@
 dispatcher:
     dummy_cache: dummy-cache
     products_url: http://www.astro.unige.ch/cdci/astrooda_
-    dispatcher_url: 0.0.0.0
+    dispatcher_host: 0.0.0.0
     dispatcher_port: 8001
```

Comment:
Actually now dispatcher_port and dispatcher_host are identical to bind_port and bind_address, and serve the same purpose. Oh, could the purpose be spelled out in the example config?

Reply:
That was the reason of my confusion yesterday